tests/instruments/rohdeschwarz/test_fsl.py

```diff
@@ -0,0 +1,79 @@
+# from time import sleep
+
+import pytest
+from pymeasure.instruments.rohdeschwarz.fsl import FSL
+
+# pytest.skip("Only works with connected hardware", allow_module_level=True)
```

Comment:
Why comment this out? It should be skipped after all.

Reply:
Resolved via [97b52bd](https://github.com/bleykauf/pymeasure/pull/4/commits/97b52bdbb7720ee70b29b289be5a6ca617880581)